exp/views/study.py

```diff
@@ -1955,6 +1955,7 @@ def get_response_values_for_demographics(self, study):
             .select_related("child", "child__user", "study", "demographic_snapshot")
```

Comment:
While we're here, can we fix the misspellings and mixed casing in the ~40 lines above? stuff like `allHeaders` -> `all_headers`, `StudyParticiapnts` -> `StudyParticipants`

Reply:
done, thanks for noticing those